tools/docker/build.sh

```diff
@@ -7,10 +7,9 @@
 ###############################################################
 
 scriptdir="$(cd "${0%/*}"; pwd)"
-rootdir="${scriptdir%/*/*}"
-
-. ${rootdir}/tools/docker/functions.sh
```

Comment:
I'm missing a description for the PR, so can't really review cause I don't know what is fixed

Reply:
This is draft yet. Will be ready by EOD